portfolio/src/main/webapp/script.js

```diff
@@ -29,3 +29,10 @@ function addRandomGreeting() {
     const greetingContainer = document.getElementById('greeting-container');
     greetingContainer.innerText = greeting;
 }
+
+/**
```

Comment:
Consider removing this empty function

(Or write it and make it not empty :) )